src/scaffolded-question/components/iframe-runtime.tsx

```diff
@@ -27,67 +27,68 @@ export const IframeRuntime: React.FC<IProps> =
   const [ hint, setHint ] = useState("");
   const iframeRef = useRef<HTMLIFrameElement>(null);
   const phoneRef = useRef<IframePhone>();
+  // Why is interativeState and setInteractiveState kept in refs? So it's not necessary to declare these variables as
+  // useEffect's dependencies. Theoretically this useEffect callback is perfectly fine either way, but since
```

Comment:
A cleaner solution -- if only it were supported -- would be an eslint comment that would make it possible to specify that a particular value is intentionally left out of the dependencies list. In the absence of that, this seems like a reasonable alternative.